doc/source/developer-guides/general.rst

```diff
@@ -11,8 +11,13 @@ New platforms can be implemented by inheriting the
 :class:`qibolab.abstract.AbstractPlatform` and implementing its abstract
 methods. In particular the developer should:
 
-* Perform an ``AbstractPlatform`` inheritance.
-* Create a yaml in ``src/qibolab/runcards/`` containing all the instruments and the calibration parameters
+* Perform an ``AbstractPlatform`` inheritance, we suggest saving it in ``src/qibolab/platforms/``, if you want to include it in the ``Qibolab`` library.
```

Comment:
To be consistent with what you have on the second point you could specify that your child class is `ExamplePlatform` and that you are coding it in the file `src/qibocal/platforms/example.py`.